src/main/java/seedu/address/model/CategoryList.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import seedu.address.model.expense.Category;
+import seedu.address.model.expense.Expense;
+
+
+//@@author Jiang Chen
+/**
+ * Manage all Category class created during addition of expense.
+ * Duplicate is not allowed.
+ * */
+public class CategoryList {
+    public final HashMap<Category, ArrayList<Expense>> categoryList;
+
+    /**
+     * Create a HashMap of {@code Category}. The key is Category itself and the value is {@code expenseList}
```

Comment:
Is this correct? Should this not be a `HashMap` of `Category` and an `ArrayList` of `Expense`